packages/components/inputs/input-utils/src/styles.ts

```diff
@@ -78,9 +79,15 @@ const getInputStyles = (props: TInputProps) => {
     display: flex;
     flex: 1;
     font-family: inherit;
-    font-size: ${designTokens.fontSize30};
-    height: ${designTokens.heightForInput};
-    min-height: ${designTokens.heightForInput};
+    font-size: ${props.isCondensed
+      ? `${designTokens.fontSize20}`
+      : `${designTokens.fontSize30}`};
```

Comment:
Seems like we are making these changes in multiple components, maybe we can have a token for the condensed state font size.
```suggestion
    font-size: ${props.isCondensed
      ? `${designTokens.fontSizeForInputWhenCondensed}`
      : `${designTokens.fontSize30}`};
```